content/docs/for-buildpack-authors/how-to/distribute-buildpacks/craft-order.md

```diff
@@ -3,8 +3,18 @@ title="Craft a buildpack order"
 weight=99
 +++
 
+Buildpack order is
```

Comment:
Probably all of this terminology from the spec could land somewhere on this page:

A buildpack is a directory containing a buildpack.toml. Buildpacks analyze application source code and determine the best way to build it.

A buildpack group, or group, is a list of one or more buildpacks that are designed to work together - for example, a buildpack that provides node and a buildpack that provides npm.

An order is a list of one or more groups to be tested against application source code, so that the appropriate group for a build can be determined.

A component buildpack is a buildpack containing /bin/detect and /bin/build executables. Component buildpacks implement the [Buildpack Interface](https://github.com/buildpacks/spec/blob/main/buildpack.md#buildpack-interface).

A composite buildpack is a buildpack containing an order definition in buildpack.toml. Composite buildpacks do not contain /bin/detect or /bin/build executables. They MUST be [resolvable](https://github.com/buildpacks/spec/blob/main/buildpack.md#order-resolution) into a collection of component buildpacks.

Reply:
Not sure why we need to repeat these on this page?

Reply:
Ah whoops, I think I got confused because I was also looking at https://github.com/buildpacks/docs/pull/739/files#diff-aa2ff8a5bc08f09ceee1f0259cb3dff812cbe771ac48c64366ded6ae5754948d

We should at least define group and order:

A buildpack group, or group, is a list of one or more buildpacks that are designed to work together - for example, a buildpack that provides node and a buildpack that provides npm.

An order is a list of one or more groups to be tested against application source code, so that the appropriate group for a build can be determined.

Reply:
added